template/src/components/pressables/BaseButton.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactElement } from 'react'
+import {
+  ActivityIndicator,
+  Insets,
+  Pressable,
+  StyleProp,
+  StyleSheet,
+  Text,
+  TextStyle,
+  View,
+  ViewStyle,
+} from 'react-native'
+import colors from '@config/ui/colors'
+import { FC } from '@utils/types'
+
+const BORDER_RADIUS = 5
+
+type RoundButtonType = 'primary' | 'secondary'
+
+export type BaseButtonPropsType = {
+  isDisabled?: boolean | null
```

Comment:
For the sake of consistency with the [RN Button](https://reactnative.dev/docs/button#disabled) this prop should be named `disabled`.

Reply:
done